add docker

nikitabel0 opened the pull request.

Commit:
add docker

yakser commented on docker-compose.yml:
```diff
@@ -0,0 +1,29 @@
+    command: uv run python -m psychohelp.main
+
+  db:
+    image: postgres:13
```
давай 17-й сразу заюзаем

yakser commented on Dockerfile:
```diff
@@ -0,0 +1,14 @@
+FROM python:3.11-slim
```
давай попробуем 3.13 затащить сразу. в pyproject.toml >=3.8 задана, но если зависимости не разломаются, то давай по максимуму новые версии будем юзать

dangannn commented on docker-compose.yml:
```diff
@@ -0,0 +1,29 @@
+version: '3.8'
```
вот это тоже уже артефакт кажется

dangannn commented on .gitignore:
```diff
@@ -63,8 +63,7 @@ Thumbs.db
 *.sublime-workspace
 
 # Docker
```
а комментик зачем остался?

Screxy commented on docker-compose.yml:
```diff
@@ -0,0 +1,29 @@
+    ports:
+      - "8000:8000"
+    environment:
+      - DATABASE_URL=postgresql+asyncpg://myuser:mypassword@db:5432/mydatabase
```
лучше вынести в ENV`myuser ` `mypassword ` `5432 ` `mydatabase `

Screxy commented on docker-compose.yml:
```diff
@@ -0,0 +1,29 @@
+      - postgres_data:/var/lib/postgresql/data
+
+volumes:
+  postgres_data:
```
postgres_data давай лучше добавим префикс проекта, чтобы не было конфликтов, на случай если у разработчика уже есть такое
например `mps_ postgres_data`

Screxy replied:
еще перенос строки нужно добавить

Screxy commented on docker-compose.yml:
```diff
@@ -0,0 +1,29 @@
+  db:
+    image: postgres:13
+    environment:
+      - POSTGRES_USER=myuser
```
вот это тоже лучше в ENV хранить

Screxy commented:
стоит еще придерживаться [Conventional Commits](https://www.conventionalcommits.org/ru/v1.0.0/)

Commit:
build: complete Docker stack upgrade

- Upgrade PostgreSQL from 13 to 17
- Upgrade Python base image from 3.11 to 3.13
- Add environment variables configuration
- Add project-prefixed volume name
